schemata/imagemeasurements.yaml

```diff
@@ -0,0 +1,216 @@
+---
+id: imagemeasurements.json
+"$schema": "http://json-schema.org/draft-04/schema"
+title: ImageMeasurements
+description: "Complex image measurements"
+definitions:
+  faces:
+    description: "Extracted faces from the image"
+    type: array
+    items:
+      description: "Bounding box of a detected face"
+      allOf:
+        "$ref": "base.json#/definitions/bbox"
+      properties:
+        "$ref": "base.json#/definitions/confidence"
+  colors:
+    description: "Extracted colors from the image, represented as an array of
+      at least 5 RGB colors"
+    type: array
+    items:
+      "$ref": "base.json#/definitions/tuplecolor"
+    minItens: 5
+  saliency:
+    description: "Extracted salient region from an image"
+    type: object
+    properties:
+      bbox:
+        "$ref": "base.json#/definitions/bbox"
+        description: "Bounding box around all extracted salient regions"
+      confidence:
+        "$ref": "base.json#/definitions/confidence"
+        description: "Confidence of detected salient regions"
+      regions:
+        description: "Extracted salient regions"
+        type: array
+        items:
+          type: object
+          properties:
+            bbox:
+              "$ref": "base.json#/definitions/bbox"
+            center:
+              "$ref": "base.json#/definitions/point"
+            radius:
+              type: number
+            polygon:
+              "$ref": "base.json#/definitions/polygon"
+      # To be deprecated:
+      polygon:
+        "$ref": "base.json#/definitions/arraypolygon"
+        description: "Polygon around the salient region (warning: to be
+          deprecated by `regions` polygon)"
+        example: [[533,280],[521,280],[491,238],[486,288],[419,289],[411,308],[360,308],[347,298],[343,289],[329,291],[343,326],[356,329],[372,370],[378,365],[421,365],[439,371],[443,360],[454,358],[454,327],[482,315],[522,315],[523,339],[530,339],[533,316],[538,310]]
+      center:
+        "$ref": "base.json#/definitions/arraypoint"
+        description: "Point localized in the center of the salient region
+          (warning: to be deprecated by `regions` center)"
+        example: [433,300]
+      radius:
+        description: "Radius of the circle around the salient region (warning:
+          to be deprecated by `regions` radius)"
+        type: number
+        example: 108.079
+      bounding_rect:
+        "$ref": "base.json#/definitions/arrayrectangle (warning: to be
+          deprecated by `regions` bbox)"
+        description: "Rectangle around the salient region"
+        example: [[329,238],[539,372]]
+  histogram:
+    description: "Histogram of color levels"
+    type: object
```

Comment:
`histogram` is an `object` with properties (`histogram.r`, `histogram.l`, ...) that are `array`s.